bin/templates/project/dependencies.gradle

```diff
@@ -28,16 +28,16 @@ repositories {
 
 dependencies {
 	// Keep in sync with lib/src/main/project.properties
-	compile 'com.android.support:appcompat-v7:23.2.1'
-	compile 'com.android.support:cardview-v7:23.2.1'
-	compile 'com.android.support:support-v4:23.2.1'
-	compile 'com.android.support:recyclerview-v7:23.2.1'
-	compile 'com.android.support:design:23.2.1'
-	compile 'com.google.android.gms:play-services-gcm:9.0.2'
+	compile 'com.android.support:appcompat-v7:27.1.0'
+	compile 'com.android.support:cardview-v7:27.1.0'
+	compile 'com.android.support:support-v4:27.1.0'
+	compile 'com.android.support:recyclerview-v7:27.1.0'
+	compile 'com.android.support:design:27.1.0'
+	compile 'com.google.android.gms:play-services-gcm:11.8.0'
```

Comment:
We should also update the SDK/buildTool versions in build.gradle in the same folder from 23 -> 27:
"
	compileSdkVersion 23
	buildToolsVersion "23.0.2"
"

They need to be the same version among SDK, buildTool and additional Android support libraries

Reply:
Not necessarily the buildToolsVersion, but definitely compileSdkVersion needs to remain in sync with the support library version.  Does aemm-lib-release.aar have the same changes?

I'm also curious what bearing the `target` property in this file has: https://github.com/adobe-marketing-cloud-mobile/aemm-android/blob/master/framework/project.properties

Reply:
@mttmllns  yes aemm-lib-release.aar has  it has the changes https://git.corp.adobe.com/dps/viewer-android/blob/release/release.2018.3.3/gradle.properties .Updated the other changes suggested too.

Reply:
Actually .. with the update to aemmbuild tools ..  getting an error that I am investigating when I run the following sequence of commands:

aemm project create sample-7.0
cd sample-7.0
aemm platform add ../aemm-android    (This dir ../aemm-android has the PR version of the branch with the changes for build tools)
aemm build android

error in this command:
aemm build android

Starting a new Gradle Daemon for this build (subsequent builds will be faster).

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> failed to find Build Tools revision 27.0.1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 8.524 secs

Reply:
ok interim update .. after adding a new google repository entry for android-27 in bin/templates/project/build.gradle it fails intially, but works after opening the sample project in Android Studio...investigating.